src/pages/signup/index.tsx

```diff
@@ -1,5 +1,225 @@
+          />
+          <TextField
+            margin="normal"
+            required
+            autoFocus
+            sx={textFieldStyle}
+            id="hourlyRate"
+            name="hourlyRate"
+            label="hourlyRate"
+            type="hourlyRate"
+            onChange={formik.handleChange}
+            error={formik.touched.hourlyRate && Boolean(formik.errors.hourlyRate)}
+            helperText={formik.touched.hourlyRate && formik.errors.hourlyRate}
+          />
+
+          <label htmlFor="file-upload">
+            <input
+              accept=".pdf"
+              id="file-upload"
+              type="file"
+              style={{ display: 'none' }}
+              onChange={handleFileUpload}
+            />
+            <Button variant="contained" component="span" style={fileUploadStyle}>
+              Upload CV
+            </Button>
+          </label>
+
+          <label htmlFor="img-upload">
+            <input
+              accept="image/*"
+              id="img-upload"
+              type="file"
+              style={{ display: 'none' }}
+              onChange={handleFileUpload}
+            />
+            <Button variant="contained" component="span" style={fileUploadStyle}>
+              Upload Image
+            </Button>
+          </label>
+        </>
+      );
+    }
+    return null;
+  };
+
+  return (
+    <Grid container component="main" sx={gridStyle}>
+      <CssBaseline />
+      <Grid style={{ height: 'fit-content', width: 'fit-content', overflow: 'hidden' }} item xs={false} sm={4} md={6}>
+        <img src={imageSrc} alt="login" className="imageLogin" style={imageStyle} />
+      </Grid>
+      <Grid
+        item
+        xs={12}
+        sm={8}
+        md={6}
+        component={Paper}
+        elevation={6}
+        square
+        style={{ height: '100vh', overflow: 'auto', overflowX: 'hidden' }}
+      >
+        <Box sx={boxStyle}>
+          <Typography component="h1" variant="h5" sx={{ ml: 5, fontSize: '30px' }}>
+            Join us
+          </Typography>
+          <Box component="form" noValidate sx={{ mt: 5 }} onSubmit={formik.handleSubmit}>
+            <RadioGroup
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+              aria-label="userType"
+              name="userType"
+              value={userType}
+              onChange={handleUserTypeChange}
+              row
+            >
+              <FormControlLabel
+                style={{ margin: '10px' }}
+                value="therapist"
+                control={<Radio />}
+                label="therapist"
+                labelPlacement="end"
+              />
+              <FormControlLabel
+                style={{ margin: '10px' }}
+                value="patient"
+                control={<Radio />}
+                label="patient"
```

Comment:
```suggestion
                label="user"
```
And I'd prefer to have user radio btn before therapist

Reply:
done